client/cmdhfmf.c

```diff
@@ -365,13 +373,21 @@ int CmdHF14AMfDump(const char *Cmd)
 	bool isOK = true;
 	for (sectorNo = 0; isOK && sectorNo < numSectors; sectorNo++) {
 		for (blockNo = 0; isOK && blockNo < NumBlocksPerSector(sectorNo); blockNo++) {
+			bool doesKeyAWork = true;
 			bool received = false;
-			for (tries = 0; tries < 3; tries++) {
+			for (tries = 0; tries < 6; tries++) {
 				if (blockNo == NumBlocksPerSector(sectorNo) - 1) {		// sector trailer. At least the Access Conditions can always be read with key A.
-					UsbCommand c = {CMD_MIFARE_READBL, {FirstBlockOfSector(sectorNo) + blockNo, 0, 0}};
-					memcpy(c.d.asBytes, keyA[sectorNo], 6);
-					SendCommand(&c);
-					received = WaitForResponseTimeout(CMD_ACK,&resp,1500);
+					if(doesKeyAWork){

```

Comment:
See comment above: At least the Access Conditions can always be read with key A.